### PR TITLE
Update the about menu

### PR DESCRIPTION
**Pull request by CannonLock**
- Remove the Speed Dial ( Loved how gorgeous it was, but isn't as practical as a menu )
- Add menu items for the support
- Remove the Duplicative pointer to github, opting for a single menu item the points to the relase notes on github instead.

### CODE REVIEW

**Comment by CannonLock** [comment]
Closes #1074

**Review by haoming29 — CHANGES_REQUESTED** [review]
LGTM except for one style jumpiness
When I toggle the question mark icon, the left margin of the website seems to change:
Screenshot 2024-06-03 at 11 11 08 AM

Screenshot 2024-06-03 at 11 11 06 AM

Notice the left margin by the graph y-axis labels, the margin shrinks by a couple pixels when I clicked the button. I'm on Chrome build 125.0.6422.141

**Comment by CannonLock** [comment]
@haoming29 I can't reproduce, and have no idea how these two elements could be interacting. The menu itself pulls up in a popover that is almost completely distinct from the items behind it. 

Do you mind me looking over your shoulder well you reproduce? I do notice that the padding on the right is likely the issue as it is slightly larger and pressing the other items together in the top image. 

336148092-e2ad7bf2-5e48-4fc3-ac6d-76ad2512fa36

**Comment by haoming29** [comment]
Looking at the source, when the portal is open, some code added 15px of `padding-right` to the `body` tag:
Screenshot 2024-06-03 at 3 27 12 PM
Before portal opens

Screenshot 2024-06-03 at 3 27 08 PM
After

**Comment by CannonLock** [comment]
@haoming29 Appreciate the screenshots, I couldn't reproduce organically but just hardcoded to test my fix. 

Note the commit comments, looks like the Popover adds the padding for no reason I could discover.

**Comment by haoming29** [comment]
@CannonLock Probably a MUI bug: https://stackoverflow.com/questions/60682426/material-ui-adds-padding-to-the-body-tag-when-a-dialog-is-opened and the fix seems to set `disableScrollLock: true`.

**Comment by haoming29** [comment]
IIRC, I had the same problem opening the menus on https://pelicanplatform.org/ where there's extra right padding when the menu is open

**Comment by CannonLock** [comment]
@haoming29 Yeah, sorry I noted that in my commit that fixes this. I just hardcode the body to not have padding. This way I don't have to adjust every menu that is created. 

https://github.com/PelicanPlatform/pelican/pull/1360/commits/a2534abacce31a767f0d0522585926a3f334fe4c

File: web_ui/frontend/app/origin/page.tsx [hunk 1]
```diff
@@ -57,7 +57,8 @@ export default function Home() {
                                         justifyContent:"center",
                                         display:"flex",
                                         bgcolor:"white",
-                                        borderRadius:2
+                                        borderRadius:2,
+                                        p: 1
```
Comment (haoming29) [inline web_ui/frontend/app/origin/page.tsx:61]: Do we want to do the same padding for the rest of pages? i.e. registry and director?

Comment (CannonLock) [reply web_ui/frontend/app/origin/page.tsx:61]: @haoming29 This page was the outlier, the others all ready have padding.

**Pull request merged**